zerver/tornado/event_queue.py

```diff
@@ -1128,6 +1133,7 @@ def process_message_update_event(
     wildcard_mention_user_ids = set(event_template.pop("wildcard_mention_user_ids", []))
     muted_sender_user_ids = set(event_template.pop("muted_sender_user_ids", []))
     all_bot_user_ids = set(event_template.pop("all_bot_user_ids", []))
+    disable_external_notifications = event_template.get("disable_external_notifications", False)
```

Comment:
Like other fields, this should probably be `event_template.pop` instead of `get` so that we don't send this unnecessary data to the client as event data later on.